definitions/ext-ups/kentik-vertiv_ups-dashboard.json

```diff
@@ -0,0 +1,225 @@
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Battery Status",
+            "layout": {
+              "column": 3,
+              "row": 1,
+              "width": 2,
+              "height": 4
+            },
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "dataFormatters": [],
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT if(latest(upsBatteryStatus) = 'batteryNormal', 0, 1) FACET upsBatteryStatus WHERE provider = 'kentik-ups'"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              },
+              "thresholds": [
+                {
+                  "alertSeverity": "WARNING",
+                  "value": 0.1
+                },
+                {
+                  "alertSeverity": "CRITICAL",
+                  "value": 0.2
+                }
+              ]
+            }
+          },
+          {
+            "title": "Battery Summary",
+            "layout": {
+              "column": 5,
+              "row": 1,
+              "width": 3,
+              "height": 4
+            },
+            "visualization": {
+              "id": "viz.billboard"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT latest(kentik.snmp.gpPwrBatteryCapacity) AS 'Battery Capacity (%)', latest(lgpPwrBatteryChargeStatus) AS 'Battery Charge Status' WHERE provider = 'kentik-ups'"
+                }
+              ],
+              "platformOptions": {
+                "ignoreTimeRange": false
+              }
+            }
+          },
+          {
+            "title": "Output Load Capacity %",
+            "layout": {
+              "column": 8,
+              "row": 1,
+              "width": 5,
+              "height": 4
+            },
+            "visualization": {
+              "id": "viz.line"
+            },
+            "rawConfiguration": {
+              "facet": {
+                "showOtherSeries": false
+              },
+              "legend": {
+                "enabled": true
+              },
+              "nrqlQueries": [
+                {
+                  "accountId": 0,
+                  "query": "FROM Metric SELECT latest(kentik.snmp.upsOutputPercentLoad) AS 'Output Load %' WHERE provider = 'kentik-ups' TIMESERIES 5 MINUTES"
```

Comment:
I would remove the 5 MINUTES from timeseries or set it to MAX as timerange can be changed by user

Reply:
same comment from the APC dashboard